databricks/sdk/core.py

```diff
@@ -736,9 +736,13 @@ class ApiClient:
     _cfg: Config
 
     def __init__(self, cfg: Config = None):
-        self._cfg = Config() if not cfg else cfg
+
+        if cfg is None:
+            cfg = Config()
+
+        self._cfg = cfg
         self._debug_truncate_bytes = cfg.debug_truncate_bytes if cfg.debug_truncate_bytes else 96
-        self._user_agent_base = cfg.user_agent
+        self._user_agent_base = cfg.user_agent if cfg.user_agent else None
```

Comment:
why user agent could be none? did you notice it being none?

Reply:
think it had to do with some bug in the previous code. removed it.